src/components/NavBar.tsx

```diff
@@ -10,56 +9,151 @@ import {
 } from "@mui/icons-material";
 import { signOut } from "next-auth/react";
 import { useSession } from "next-auth/react";
+import * as React from "react";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import CssBaseline from "@mui/material/CssBaseline";
+import Divider from "@mui/material/Divider";
+import Drawer from "@mui/material/Drawer";
+import IconButton from "@mui/material/IconButton";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemButton from "@mui/material/ListItemButton";
+import MenuIcon from "@mui/icons-material/Menu";
+import Toolbar from "@mui/material/Toolbar";
 
-const NavBar = () => {
-  const { status } = useSession();
+const drawerWidth = 240;
 
-  return (
-    <>
-      {status === "authenticated" && (
-        <nav className={$navbar.sidebar}>
-          <Link href="/">
-            <Image
-              src="/img/compass-logo-white.svg"
-              alt="logo"
-              className={$navbar.logo}
-              width={64}
-              height={64}
-              priority
-            />
-          </Link>
-          <br />
+export default function NavBar() {
+  const [mobileOpen, setMobileOpen] = React.useState(false);
+
+  const handleDrawerToggle = () => {
+    setMobileOpen(!mobileOpen);
+  };
 
-          <div className={$navbar.linkContainer}>
-            <Link href="/students" className={$navbar.link}>
+  const drawer = (
+    <div className={$navbar.sidebar}>
+      <Toolbar />
+      <Divider />
+      <Link href="/">
+        <Image
+          src="/img/compass-logo-white.svg"
+          alt="logo"
+          className={$navbar.logo}
+          width={64}
+          height={64}
+          priority
+        />
+      </Link>
+      <List className={$navbar.list}>
+        <ListItem disablePadding className={$navbar.linkItem}>
+          <Link href="/students" className={$navbar.link}>
+            <ListItemButton>
```

Comment:
Yes thank you for noticing that!! For now I actually just removed the listitembutton tag since all it was providing was styling, not sure if that's good for now or if we should do it the way you proposed (having the links within the button components)? @jonahchoi

Reply:
Sounds good for now!